docs/conf.py

```diff
@@ -430,7 +431,7 @@ def __getattr__(cls, name):
 
 # Insert a link to download the IPython notebook
 nbsphinx_prolog = r"""
-{% set docname = env.doc2path(env.docname, base='docs') %}
+{% set docname = "docs/" ~ env.doc2path(env.docname, base=false) %}
```

Comment:
why this change? in any case, the `/` should be replaced with `os.path.sep`

Reply:
sphinx was showing a deprecation warning that strings will no longer be accepted for `base`. I didn't know I could use `os.path` inside a jinja template.

Reply:
ok.  lets see this later then